server/models/schema.sql

```diff
@@ -1856,6 +1856,17 @@ CREATE TABLE IF NOT EXISTS `voucher_item` (
 
 -- stock tables
 
+DROP TABLE IF EXISTS `stock_setting`;
+CREATE TABLE `stock_setting` (
```

Comment:
Excellent! You'll need to add this to `migrate.sql` when you are ready.

Reply:
Done.